packages/renderer/src/lib/preferences/docker-compat/PreferencesDockerCompatibilitySocketMappingStatus.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
+import { Tooltip } from '@podman-desktop/ui-svelte';
+import { onMount } from 'svelte';
+import Fa from 'svelte-fa';
+import { router } from 'tinro';
+
+import type { DockerSocketMappingStatusInfo } from '/@api/docker-compatibility-info';
+
+import Label from '../../ui/Label.svelte';
+import ProviderInfoCircle from '../../ui/ProviderInfoCircle.svelte';
+
+let isMac = $state(false);
+let isWindows = $state(false);
+let isLinux = $state(false);
+
+let dockerSocketMappingStatusInfo: DockerSocketMappingStatusInfo | undefined = $state(undefined);
+
+let engineType: 'kubernetes' | 'podman' | 'docker' | undefined = $state(undefined);
+
+async function refreshSocketMappingStatus(): Promise<void> {
+  dockerSocketMappingStatusInfo = await window.getSystemDockerSocketMappingStatus();
+
+  if (dockerSocketMappingStatusInfo?.serverInfo?.type === 'podman') {
+    engineType = 'podman';
+  } else if (dockerSocketMappingStatusInfo?.serverInfo?.type === 'docker') {
+    engineType = 'docker';
+  } else {
+    engineType = undefined;
+  }
+}
+
+onMount(async () => {
+  const platform = await window.getOsPlatform();
+
+  isMac = platform === 'darwin';
+  isLinux = platform === 'linux';
+  isWindows = platform === 'win32';
+
+  // ask once to get the result
+  await refreshSocketMappingStatus();
+});
+</script>
+
+<div
+  class="bg-[var(--pd-invert-content-card-bg)] rounded-md mt-2 ml-2 divide-x divide-gray-900 flex flex-col lg:flex-row">
```

Comment:
gray-900 (here and below) should be replaced with something from the color registry.

Reply:
it's how the card are being displayed today:
https://github.com/containers/podman-desktop/blob/4681eecd2a536bcb9c4978f54815744f8d075a2a/packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte#L376

Reply:
solved